ee/maintained-apps/outputs/notion/windows.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
When we have blanks like this we should catch that and skip outputting the file. Checking the app manifest struct against the zero value of the same thing and logging/continuing is sufficient.

Reply:
sgtm!